db/schema.rb

```diff
@@ -10,10 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+
 ActiveRecord::Schema.define(version: 20170920095341) do
 
-  # These are extensions that must be enabled in order to support this database
+   # These are extensions that must be enabled in order to support this database
```

Comment:
Watch out for random accidental whitespace changes.  There's one above, as well.